selendroid-standalone/src/test/java/io/selendroid/standalone/android/impl/DefaultAndroidEmulatorTests.java

```diff
@@ -23,15 +23,19 @@
 import org.apache.commons.io.FileUtils;
 import org.junit.Assert;
 import org.junit.Test;
+import org.junit.Ignore;
 import org.openqa.selenium.Dimension;
 
 public class DefaultAndroidEmulatorTests {
+
+  @Ignore("Fail. Requires list of available avds")
```

Comment:
this should only fail if you have not defined any avd's on your system? if you create one, should be ok?